includes/compat/class-log.php

```diff
@@ -251,11 +253,16 @@ public function file_download_log_get_post_meta( $value, $object_id, $meta_key,
 						$key = 'api_key';
 						break;
 					case 'user':
-						$key = 'user_id';
+						$key = 'customer_id';
```

Comment:
This should go back to `user_id`. `user` and `user_id` are equivalent, but `customer_id` is separate.